src/main/resources/com/sap/oss/phosphor/fosstars/advice/oss/OssAdvice.yml

```diff
@@ -56,6 +56,30 @@ If a project runs Bandit scan checks for commits:
         url: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-a-list-of-events
       - name: An eample to trigger Bandit scan check on every pull-request.
         url: https://github.com/TNLinc/CV/blob/main/.github/workflows/bandit.yml#L3
+If a project runs GoSec scans:
+  - advice: >
+      You can open a pull request to run GoSec scans in the project using GitHub action workflow.
+    links:
+      - name: GitHub workflow action job config to run GoSec code scanning for a repository.
+        url: https://github.com/securego/gosec#github-action
+      - name: An example to run GoSec scan check as part of GitHub action workflow.
+        url: https://github.com/gofiber/fiber/blob/master/.github/workflows/security.yml#L10
+If a project runs GoSec scans with rules:
+  - advice: >
+      You can run GoSec scans with default rules.
+    links:
+      - name: GitHub workflow action job config to run GoSec code scanning for a repository.
+        url: https://github.com/securego/gosec#github-action
+      - name: Configuration to run the GoSec scan in default mode.
+        url: https://github.com/securego/gosec#usage
+If a project runs GoSec scan checks for commits:
+  - advice: >
+      You can open a pull request to trigger GoSec scans job in the project using GitHub action workflow for every pull-request.
+    links:
+      - name: GitHub workflow action config to run GoSec code scanning job on every PR of a project.
+        url: https://github.com/securego/gosec#github-action
+      - name: An example to trigger GoSec scan check on every pull-request.
+        url: https://github.com/gofiber/fiber/blob/master/.github/workflows/security.yml#L10
```

Comment:
Yes, it does. What do you suggest? Can I keep only If a project runs GoSec scan checks for commits and remove If a project runs GoSec scans with rules?

Reply:
It makes sense to keep two advices rather than three here

Reply:
I would suggest keep the advice simple that it would be good to have SecGo as analysis step with PR, yes initiating it with PR would be great. That should be the advice.